rest-api-spec/build.gradle

```diff
@@ -54,6 +54,28 @@ tasks.named("precommit").configure {
   dependsOn 'enforceYamlTestConvention'
 }
 
-tasks.named("yamlRestCompatTestTransform").configure({ task ->
-  task.skipTest("tsdb/140_routing_path/multi-value routing path field", "Multi-value routing paths are allowed now. See #112645")
-})
+tasks.named("yamlRestCompatTestTransform").configure(
```

Comment:
I notice this change is being backported to 8.16. If that's the case, could these exception be removed once that is complete?

Reply:
Correct, these skips are merely to get this submitted green. Will remove once the backport is complete.